File: outros/ws - pycharm/primeiro/soma_vetor.py
n = int(input("Quantos números será digitado: "))

a: [float] = [0 for x in range(n)]

for i in range(0, n):
    a[i] = float(input("Escreva um número: "))

soma = 0
print("\nValores: ", end="")
for i in range(0, n):
    print(a[i], " ", end="")
    soma = soma + a[i]

media = soma / n

print(f"\nSoma = {soma:.2f}")
print(f"Media = {media:.2f}")

File: outros/eclipse/exercicios/src/soma_vetor.java
import java.util.Locale;
import java.util.Scanner;

public class soma_vetor {
	public static void main(String[] args) {
		 Locale.setDefault(Locale.US);
		 Scanner sc = new Scanner(System.in); 
		 
		 System.out.print("Quantos números você vai digitar: ");
		 int n = sc.nextInt();
		 
		 double[] a = new double[n];
		 
		 for (int i = 0; i < n; i++) {
			 System.out.print("Digite um número: ");
			 a[i] = sc.nextDouble();
		 }
		 
		 double soma = 0;
		 System.out.print("Valores = ");
		 for (int i = 0; i < n; i++) {
			 System.out.print(String.format("%.1f",a[i]) + " ");
			 soma = soma + a[i];
		 }
		 
		 System.out.print("\nSoma = " + String.format("%.2f",soma));
		 
		 double media = soma / n;
		 System.out.println("\nMédia = " + String.format("%.2f", media));
		 
		 sc.close();		 
		 
	}
}